src/components/HoverArrow.tsx

```diff
@@ -0,0 +1,36 @@
+import { token } from '../../styled-system/tokens/index.mjs'
+
+interface HoverArrowType {
+  hoverArrowVisibility: 'above' | 'below' | null
+  scrollTop: number
+  arrowBottom: number
+}
+
+/** Renders upward/downward arrow when hovering over a sorted context. */
+const HoverArrow = ({ hoverArrowVisibility, scrollTop, arrowBottom }: HoverArrowType) => {
+  return (
+    hoverArrowVisibility && (
+      <div
+        style={{
+          width: '0',
+          height: '0',
+          borderLeft: '10px solid transparent',
+          borderRight: '10px solid transparent',
+          position: 'absolute',
```

Comment:
Why `position: absolute` instead of `position: fixed`? It seems like `position: absolute` would involve a lot more manual calculation to get it in the viewport, while `position: fixed` does that for you.